src/view/components/toolbar/motion/Accelerometer.tsx

```diff
@@ -44,12 +46,38 @@ interface IProps {
         Z_AXIS: number;
     };
     onUpdateValue: (sensor: SENSOR_LIST, value: number) => void;
+    onSelectGestures?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+    onSendGesture?: (isActive: boolean) => void;
 }
+
+const GESTURE_BUTTON_MESSAGE = "Send Gesture";
+
 export const Accelerometer: React.FC<IProps> = (props: IProps) => {
     return (
         <div className="AccelerometerBar">
             <br />
             {/* Implement Gestures Here */}
+            <Dropdown options={GESTURES} onSelect={props.onSelectGestures} />
+            <SensorButton
+                label={GESTURE_BUTTON_MESSAGE}
+                onMouseDown={() => {
+                    if (props.onSendGesture) {
+                        props.onSendGesture(true);
+                    }
+                }}
+                onMouseUp={() => {
+                    if (props.onSendGesture) {
+                        props.onSendGesture(false);
+                    }
+                }}
+                onKeyDown={(e: React.KeyboardEvent) => {
+                    handleOnKeyDown(e, props.onSendGesture);
+                }}
+                onKeyUp={(e: React.KeyboardEvent) => {
+                    handleOnKeyUp(e, props.onSendGesture);
+                }}
+                type="gesture"
+            />
             <ThreeDimensionSlider
                 axisProperties={MOTION_SENSOR_PROPERTIES}
                 onUpdateValue={props.onUpdateValue}

```

Comment:
That is some good feedback, do you think if we make the instruction clearer it would help?
Maybe something like: "Select a gesture then press the button to send the gestures or you could change acceleration manually with the sliders."

Reply:
To be honest, it's very tricky and I am not super familiar with UX design. I would consult someone who would be more familiar like the design interns or Katelynn.
If I had to come up with something (keeping in mind our timeline), I think maybe separating the gestures from the accelerometer sliders would clear up things. Up to you.

Reply:
Ok understandable. What I'm thinking of doing with the time constraints is separate in the same modal the manual acceleration and sending gestures. Separating the 2 sensors would require extra work and svgs that we don't currently have.